build.rs

```diff
@@ -524,6 +524,16 @@ fn obj_path(out_dir: &Path, src: &Path, obj_ext: &str) -> PathBuf {
     out_path
 }
 
+#[allow(box_pointers)]
+fn remap_debug_prefix() -> &'static str {
+    use std::env;
+    let cwd_path = env::current_dir().expect("Couldn't get current directory");
+    let mut flag = String::from("-fdebug-prefix-map=");
+    flag.push_str(cwd_path.to_str().expect("Invalid path"));
+    flag.push_str("=.");
+    Box::leak(flag.into_boxed_str())
```

Comment:
I don't understand why `Box::leak` is required here. I suggest we just return a `String`.

Reply:
Done.